script.js

```diff
@@ -0,0 +1,245 @@
+textarea.className = 'textarea';
+textarea.setAttribute('autofocus', 'true');
+
+document.body.appendChild(div);
+div.append(textarea);
+div.append(keyBoard);
+
+// add rows on keyboard
+for (let j = 0; j < keyboard.length; j++) {
+
+  const divRow = document.createElement('div');
+  divRow.className = 'row';
+
+  // add keybutons on keyboard in row
+  for (let i = 0; i < keyboard[j].length; i++) {
+    const divButton = document.createElement('div');
+    divButton.className = 'keybutton';
+
+    // change style for keyButtons
+    switch (keyboard[j][i].name) {
+      case 'backsp':
+        divButton.className += ' backsp';
+        break;
+      case 'tab':
+        divButton.className += ' tab';
+        break;
+      case 'caps':
+        divButton.className += ' caps';
+        break;
+      case 'enter':
+        divButton.className += ' enter';
+        break;
+      case 'shift':
+        divButton.className += ' shift';
+        break;
+      case 'ctrl':
+        divButton.className += ' ctrl';
+        break;
+      case 'fun':
+        divButton.className += ' fun';
+        break;
+      case 'alt':
+        divButton.className += ' alt';
+        break;
+      case 'space':
+        divButton.className += ' space';
+        break;
+    }
+
+    // add on HTML document text on buttons
+    if (keyboard[j][i].length <= 2) {
+      divButton.innerHTML += `<span class="text" data-key="${keyboard[j][i][0]}">${keyboard[j][i][0]}</span><span class="textup" data-key="${keyboard[j][i][1]}">${keyboard[j][i][1]}</span>`;
+    } else {
+      divButton.innerHTML += `<span class="functionButton" data-key="${keyboard[j][i].key}">${keyboard[j][i].name}</span>`;
+    }
+    divRow.append(divButton);
+  }
+  keyBoard.append(divRow);
+}
+
+
+// hover buttons on shift
+textarea.addEventListener('keydown', function (event) {
+  if (event.key != 'Shift') return;
+  text.forEach((el) => el.style.display = 'none');
+  textUp.forEach((el) => el.style.display = 'inline');
+});
+
+textarea.addEventListener('keyup', function (event) {
+  if (event.key != 'Shift') return;
+  text.forEach((el) => el.style.display = 'inline');
+  textUp.forEach((el) => el.style.display = 'none');
+
+});
+
+// hover buttons on virtual keybord
+textarea.addEventListener('keydown', () => {
+  const element = document.querySelector('[data-key="' + event.key + '"]');
+  element.parentElement.classList.add('hover');
+
+ if (event.key === 'Tab') {
+   textarea.value += "\t";
+   event.preventDefault()
+ } else if (event.key === 'Enter') {
+   textarea.value += "\n";
+ }
+});
+
+textarea.addEventListener('keyup', () => {
+  const element = document.querySelector('[data-key="' + event.key + '"]');
+  element.parentElement.classList.remove('hover');
+});
+
+let text = document.querySelectorAll('.text');
+let textUp = document.querySelectorAll('.textup');
+text.forEach((el) => el.style.display = 'none');
+textUp.forEach((el) => el.style.display = 'inline');
```

Comment:
Use use these lines everywhere, move to function `changeCase` or something like that